addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1060,28 +1077,26 @@ options.registry.menu_data = options.Class.extend({
                         },
                     });
                 }},
-                {text: _t("Edit the menu"), classes: 'btn-primary', click: function () {
-                    this.trigger_up('action_demand', {
-                        actionName: 'edit_menu',
-                        params: [
-                            function () {
-                                var prom = new Promise(function (resolve, reject) {
-                                    self.trigger_up('request_save', {
-                                        onSuccess: resolve,
-                                        onFailure: reject,
-                                    });
-                                });
-                                return prom;
-                            },
-                        ],
-                    });
-                }},
+                {text: _t("Edit the menu"), classes: 'btn-primary', click: () => {this.editMenuCallBack(this.$target)}, close: true},
```

Comment:
It was not close: true before and I think it is better that way no?

Reply:
It was a request from sbu to close the first menu once we open the edit menu.

Reply:
It was a request from Sbu to close it once we open the edit menu.

Reply:
Ok, I thought it was the edit menu dialog itself, my bad